apps/web/playwright/change-username.e2e.ts

```diff
@@ -7,68 +7,53 @@ import { MembershipRole } from "@calcom/prisma/enums";
 import { moveUserToOrg } from "@lib/orgMigration";
 
 import { test } from "./lib/fixtures";
-import { IS_STRIPE_ENABLED } from "./lib/testUtils";
+import { IS_STRIPE_ENABLED, submitAndWaitForResponse } from "./lib/testUtils";
 
 test.describe.configure({ mode: "parallel" });
 
 const IS_SELF_HOSTED = !(
   new URL(WEBAPP_URL).hostname.endsWith(".cal.dev") || !!new URL(WEBAPP_URL).hostname.endsWith(".cal.com")
 );
 
+const TESTING_USERNAMES = [
+  {
+    username: "demousernamex",
+    description: "",
+  },
+  {
+    username: "demo.username",
+    description: " to include periods(or dots)",
+  },
+];
```

Comment:
Merged tests into parametized one since it was the exact same code with only username differences